### ISSUE TITLE
Changelog link in web UI isn't a link

### ISSUE THREAD

**Issue by mlucy** [rethinkdb/rethinkdb#4050]
This might be intentional, but if you have a 1.16.2 server the message in the web UI saying that 1.16.3 is available looks like this:

```
A newer version of the RethinkDB server is available: 1.16.3. You can read the changelog at <https://github.com/rethinkdb/rethinkdb/releases>.
```

But the link isn't clickable, so you have to copy-paste it into your URL bar.  (Also, why are we quoting with `<`/`>` characters?)

**Comment by VeXocide** [comment]
It's common to quote a link with `<`/`>` since they can't appear in a URL unencoded, where a `.` may for example. This way it's clear the URL is `<https://github.com/rethinkdb/rethinkdb/releases>` and not `<https://github.com/rethinkdb/rethinkdb/releases.>`.

**Comment by timmaxw** [comment]
See [here](http://peterkaminski.com/wiki/Angle_Bracket_URLs) for more information.

**Comment by danielmewes** [comment]
I agree that we should make it clickable, though it's not very important.

Assigning @deontologician

**Comment by danielmewes** [comment]
This already seems to be working fine in newer versions.

![update notification](https://cloud.githubusercontent.com/assets/505365/7261602/00aba9ae-e828-11e4-8962-fb77c9311172.png)

**Comment by danielmewes** [comment]
Oh wait, I think @mlucy you meant the message that appears in the server log?

We could make it clickable by detecting URLs in log messages, but I think it's not important and not worth doing now.
